packages/components/src/Mobile.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Zoom from '@mdx-deck/components/src/Zoom'
```

Comment:
This should use the named import instead of digging into the internal directory structure of the module

Reply:
Sorry, I misread the line, this should be a local import.

```suggestion
import Zoom from './Zoom'
```